torchvision/models/__init__.py

```diff
@@ -15,4 +15,8 @@
 from .swin_transformer import *
 from .maxvit import *
 from . import detection, optical_flow, quantization, segmentation, video
-from ._api import get_model, get_model_builder, get_model_weights, get_weight, list_models
+
+# We're making Weights and WeightsEnum public for packages like torchgeo who are
+# interested in using them https://github.com/pytorch/vision/issues/7094
```

Comment:
Not sure if that is required as a code comment? I mean we don't explain for all the other things as well. Aren't we just taking them as part of the public API now?

Reply:
They're sort of special in the sense that they're more developer tools intended to be used by downstream libraries, rather than your typical transform / model which are made for end users.
I don't think we have ever made any developer tool public yet, and I added the comment to indicate that this is not done by mistake. I won't fight to keep it though :)

Reply:
Good point, I didn't consider that. Maybe change the wording to something like "These are public ..."? My initial understanding was that this was meant as a comment for the reviewer or the PR not someone arbitrary looking at the code.

Reply:
I updated the comment, hopefully the intent is clear now!

Reply:
Yes, I like the wording better now.